probe-rs/src/debug/snapshots/probe_rs__debug__debug_info__test__unwinding_in_exception_trampoline.snap

```diff
@@ -8,10 +8,10 @@ Frame:
   directory: /home/dominik/code/probe-rs/probe-rs-repro/nrf/exceptions/src
   file: Some("main.rs")
   line: Some(22)
-  column: Some(LeftEdge)
- frame_base:      Some(2001fff0)
+  column: Some(Column(1))
+ frame_base:      Some(2001ffc8)
 Frame:
- function:        SVCall
+ function:        Supervisor call.
```

Comment:
Why is this full text now? I believe the function is actually named SVCall?

Reply:
The intention was to make them a bit more descriptive/useful, as in what would happen when for example, you would get something like `HardFault. Cause: Escalated BusFault (Precise data access error) at location: 0x3ffffffc.`

In hindsight, I think calling this case `SVCall` is actually more accurate,  so I'm going to revert to that naming.

Reply:
@Yatekii In looking at the various Arm references, e.g. [Armv8-M](https://developer.arm.com/documentation/107706/0100/Exceptions-and-interrupts-overview/Exception-types?lang=en), we have a couple of options here, and if I am going to change these, I think some consistency is called for:

1. Keep them "descriptive", i.e. as in current code. The unwind actually has frames for the `__cortex_m_rt_SVCall` and `__cortex_m_rt_SVCall_trampoline` functions, so we are not usurping those. The actual function frames remain intact in our unwind, and this handler frame is there only to ensure we don't loose the register values at the time of invoking those frames.
2. Name them by the spec fault name, in which case the `SVCall` in this example will become `SVC`
3. Name them by the spec "Exception handler name in CMSIS-Core", in which case the `SvCall` in this example becomes `SVC_Handler`
4. Name them based on `cortex_m_rt` convention (not always consistent with the spec), that uses an expanded name like `SVCall`.

What are your thoughts?

Reply:
I think 2. and 4. appeal to me most. But one thing is for sure: the `.` in the string has to go :)

Reply:
Thanks. I can align on #2 :)

Reply:
Done :)